tests/test_external_account_authorized_user.py

```diff
@@ -0,0 +1,129 @@
+import datetime
+import json
+
+import mock
+import pytest  # type: ignore
+from six.moves import http_client
+
+from google.auth import exceptions
+from google.auth import external_account_authorized_user
+from google.auth import transport
+
+
+class TestCredentials(object):
+    TOKEN_URL = "https://sts.googleapis.com/v1/token"
+    TOKEN_INFO_URL = "https://sts.googleapis.com/v1/introspect"
+    REVOKE_URL = "https://sts.googleapis.com/v1/revoke"
+    PROJECT_NUMBER = "123456"
+    POOL_ID = "POOL_ID"
+    PROVIDER_ID = "PROVIDER_ID"
+    AUDIENCE = (
+        "//iam.googleapis.com/projects/{}"
+        "/locations/global/workloadIdentityPools/{}"
+        "/providers/{}"
+    ).format(PROJECT_NUMBER, POOL_ID, PROVIDER_ID)
+    REFRESH_TOKEN = "refreshtoken"
```

Comment:
lets be consistent with the cases here e.g. REFRESH_TOKEN

Reply:
Done